adage/adage/config.py.template

```diff
@@ -217,5 +217,29 @@ CIRCLECI_CONFIG.update({
     },
 })
 
+DOCKER_DEV = CIRCLECI_CONFIG.copy()
+DOCKER_DEV.update({
+    'databases': {
+        'default': {
+            'ENGINE': 'django.db.backends.postgresql_psycopg2',
+            # database name and user cannot have upper case letters
+            'NAME': 'docker_adage',
+            'USER': 'docker_adage',
+            'PASSWORD': 'password123',
```

Comment:
So do we expect the users to customize this database password? If yes, we may want to add some comments here.

Reply:
I think this is actually more of a dummy, non-secure password for the development docker database. I was thinking it would not be changed by the user. What do you think @dongbohu and @mhuyck?

Reply:
If it is just a development database, I think this dummy password is acceptable.

Reply:
As long as there is little chance that this configuration would inadvertently be used for deploying anywhere, I think it's fine. I like that this config is explicitly labeled as `DOCKER_DEV`, which helps with that.